projects/components/src/table/table.component.scss

```diff
@@ -71,7 +71,7 @@
   }
 
   .header-column-resize-handle {
-    padding: 6px 2px;
+    padding: 6px 0px;
```

Comment:
doesn't this put the width of the resize handle down to 1px? I think it would make it very hard to grab. The screenshots look good but can we accomplish the same thing by reducing the header padding rather than the resize handles?

Reply:
done